Changes

```diff
@@ -30,6 +30,9 @@ Revision history for Perl extension App::Sqitch
        Exasol and because `FAILURE` maps to exit code `1`, which has in the
        past been more akin to a warning from Sqitch. Thanks to @vectro for the
        report (#831).
+     - Set all text column collations to "POSIX" on Postgres 9.1 and higher.
+       and Yugabyte 2.9 and higher. Does not apply to existing registries,
+       only new ones. Thanks to @datafoo for the suggestion (#763)!
```

Comment:
Doesn't this also apply when you upgrade the registry, at least for some columns?

Reply:
It would if we added more upgrades, that's true.